container_pipeline/workers/linter.py

```diff
@@ -81,15 +81,37 @@ def lint(self):
             out, err = run_cmd_out_err(command)
             if err == "":
                 response = self.handle_lint_success(out)
+                self.logger.info(
+                    "Docker file linting successful going for job creation")
```

Comment:
We're not going for job creation here. We're doing OpenShift project creation.

Reply:
done,